harvester/DBInterface.py

```diff
@@ -38,22 +37,22 @@ def __init__(self, params):
 
 			# Determine if the database schema needs to be updated
 			dbversion = self.get_db_version()
-			files = os.listdir("sql/" + self.dbtype + "/")
+			files = os.listdir("sql/" + str(self.dbtype) + "/")
```

Comment:
Similarly to the above -- this was working previously as far as I know, so this string recasting is just to be cleaner?

Reply:
I was accounting for the possibility that someone put a non-string value into what should be a string in the config file, which would break the code